packages/gafl-webapp-service/src/__tests__/server.spec.js

```diff
@@ -24,6 +29,10 @@ export const catboxOptions = {
   ]
 }
 
+afterEach(async () => {
+  await server.stop()
```

Comment:
I don't think you need to await this; and if you remove the await, you don't have to make the function async.